sdk/jobs/pipelines/1c_pipeline_with_hyperparameter_sweep/pipeline_with_hyperparameter_sweep.ipynb

```diff
@@ -79,31 +80,21 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# 2. Create command component\n",
-    "We defined a command component as trial component in [dsl_component.py](dsl_component.py), which trains a neural network for MNIST image classification with TensorFlow. The component function is train.\n"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "%load_ext autoreload\n",
-    "%autoreload 2\n",
+    "# 2. Pipeline job with hyperparameter sweep\n",
+    "Similar to [1b_pipeline_with_python_function_components](../1b_pipeline_with_python_function_components/), we define 3 sample components using dsl.command_component in [dsl_component.py](dsl_component.py) first and then build the pipeline.\n",
```

Comment:
I use dummy components here to make the pipeline as simple as possible as it's a 1c example.